src/components/sync-dialog/SyncDialog.jsx

```diff
@@ -2,19 +2,18 @@ import React from "react";
 import i18n from "@dhis2/d2-i18n";
```

Comment:
On Sync error, the button [synchronize] remains disabled:

![image](https://user-images.githubusercontent.com/24643/62850420-148e9f00-bce3-11e9-9e5d-49d384e4421b.png)

Btw, the error was that I kept a trailing "/" on the instance URL, maybe we should clean it up on save?

Reply:
```
    public setUrl(url: string): Instance {
        return new Instance({ ...this.data, url: url.replace(/\/+$/, "") });
    }
```